src/algebra/pointwise.lean

```diff
@@ -504,7 +504,7 @@ Inf_le $ λ x ⟨s, t, hs, ht, hx⟩, hx ▸ (closure S ⊔ closure T).mul_mem
     (set_like.le_def.mp le_sup_right $ subset_closure ht)
 
 @[to_additive]
-lemma sup_eq_closure (H K : submonoid M) : H ⊔ K = closure (H * K) :=
+lemma sup_eq_closure (H K : submonoid M) : H ⊔ K = closure (H * K : set M) :=
```

Comment:
It's quite unfortunate that lean now needs this cast. It would be good to know how often this comes up. Probably the best way to find out would be to do `submodule.span` too, since that is used far more than `submonoid.closure`.

Reply:
Yeah, I can't figure out why this cast is suddenly now needed.